core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -216,14 +239,25 @@ private static PendingIntent createMediaButtonIntent(Context context, int eventC
     }
 
     private static String getProgressString(int position, int duration, float speed) {
+        showTimeLeft = UserPreferences.shouldShowRemainingTime();
         if (position >= 0 && duration > 0) {
             TimeSpeedConverter converter = new TimeSpeedConverter(speed);
+            int remainingTime = converter.convert(Math.max(duration - position, 0));
             position = converter.convert(position);
             duration = converter.convert(duration);
-            return Converter.getDurationStringLong(position) + " / "
-                    + Converter.getDurationStringLong(duration);
+
+            if (showTimeLeft) {
+                return Converter.getDurationStringLong(position) + " / " +  ((remainingTime > 0) ? "-" : "")
+                        + Converter.getDurationStringLong(remainingTime);
+            }else{
+                return Converter.getDurationStringLong(position) + " / "
+                        + Converter.getDurationStringLong(duration);
+            }
+
         } else {
             return null;
         }
     }
+
```

Comment:
Could you remove these 2 extra white spaces?

Reply:
@tonytamsf Sure